src/auto-lightbox.js

```diff
@@ -27,7 +27,11 @@ export const AutoLightboxEvents_Enum = {
 export function installAutoLightboxExtension(ampdoc) {
   const {win} = ampdoc;
   // Only enabled on single documents tagged as <html amp> or <html ⚡>.
-  if (!isAmphtml(win.document) || !ampdoc.isSingleDoc()) {
+  if (
+    !isAmphtml(win.document) ||
+    !ampdoc.isSingleDoc() ||
+    win.document.body.hasAttribute('data-amp-auto-lightbox-disable')
```

Comment:
When this is called, we can't guarantee that the `<body>` (or all of its attributes) have bene parsed. It would be safer to put this on the `<html>` element instead.

Reply:
@jridgewell Should it be something like this:
```html
<html ⚡ lang="en" data-amp-auto-lightbox-disable>
...
</html>
```

Please correct me if there misinterpretation.

Reply:
Yes, that's correct.

Reply:
Ok. Let me revise implementation 👍🏻

Reply:
@jridgewell Done. I have revised implementation and updated markdown (refer https://github.com/ampproject/amphtml/pull/37854/commits/02ff8d1337704b0ea3cb2c7046ff90368684ce46)

Let me know if there is anything that needs to be taken care of.